src/components/Layout/Collapsible/Collapsible.js

```diff
@@ -6,17 +6,26 @@ const Collapsible = ({
   children,
   contentMargin,
   collapsableDivClassName,
+  collapsableHeadingClassName,
 }) => {
   const [open, setOpen] = useState(true)
 
+  const collapsableHeading = () => {
+    return collapsableHeadingClassName == 'h3' ? (
+      <h3 className="lbh-heading-h3 lbh-collapsible__heading">{heading}</h3>
+    ) : (
+      <h2 className="lbh-heading-h2 lbh-collapsible__heading">{heading}</h2>
+    )
+  }
+
```

Comment:
Why do we need this ?
And could we not use cx to conditionally render classes instead to make it a bit nicer